tests/unit/libbpf_test.cpp

```diff
@@ -3771,3 +3771,68 @@ _utility_test(ebpf_execution_type_t execution_type)
 }
 
 DECLARE_ALL_TEST_CASES("utility_test", "[libbf]", _utility_test);
```

Comment:
```suggestion
DECLARE_ALL_TEST_CASES("utility_test", "[libbpf]", _utility_test);
```
Not part of your PR but I just spotted this so please fix either in this PR or in a separate one :)